tools/IoTEdgeDevOps/DevOpsLibTest/BuildManagementTest.cs

```diff
@@ -35,7 +35,7 @@ public void TearDown()
         }
 
         [Test]
-        public async Task TestGetLatestBuildsAsyncWithNoDefinitionId()
+        public void TestGetLatestBuildsAsyncWithNoDefinitionId()
         {
             ArgumentException ex = Assert.ThrowsAsync<ArgumentException>(
```

Comment:
Does `Assert.ThrowsAsync<ArgumentException>()` has an implicit await for the await-task you are passing in?

Reply:
I guess so.  I changed this method to void as VS2019 gives warning/suggestion.  And I ran all tests and all passed.